app/models/pet_applicant.rb

```diff
@@ -1,4 +1,9 @@
 class PetApplicant < ApplicationRecord
   belongs_to :pet
   belongs_to :applicant
+
+  def self.find_by_parents(pet_id, applicant_id)
+    #potentially use find_by
+    PetApplicant.find_by(pet_id: pet_id, applicant_id: applicant_id)
```

Comment:
I might be missing something here but I'm unsure of two things...Why is this a class method and not an instance method when it seems to be used to find an instance of PetApplicant...Also is this necessary? It seems like you should just be able to call pet.applications to find all applications that belong to that pet.

Reply:
So I'm trying to find a specific instance of a pet and it's applicant. For example: an application may have many pets and a pet may be attached to many applications. I need to find one instance of a pet connected to a specific application.